src/app/modules/ix-feedback/feedback-dialog/feedback-dialog.component.html

```diff
@@ -0,0 +1,52 @@
+<h1 matDialogTitle>{{ 'How would you rate this page?' | translate }}</h1>
+<mat-dialog-content>
+  <button
+    mat-icon-button
+    mat-dialog-close
+    class="close-feedback-dialog"
+    ixTest="close-feedback-dialog"
+  >
+    <ix-icon name="clear"></ix-icon>
+  </button>
+
+  <form [formGroup]="form">
+    <ix-star-rating
+      formControlName="rating"
+      [label]="'Select rating' | translate"
+      [required]="true"
+    ></ix-star-rating>
+
+    <ix-textarea
+      formControlName="message"
+      [label]="'Message' | translate"
+      [rows]="5"
+      [placeholder]="'Share your thoughts on our product\'s features, usability, or any suggestions for improvement.' | translate"
+    ></ix-textarea>
+  </form>
+
+  <div class="file-ticket-helper">
```

Comment:
Let's move this to the top of the dialog. Also `File Ticket` doesn't look good as button, let's make it a link.